### PR TITLE
test new ssr approach

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Fixes: https://github.com/JoviDeCroock/hooked-head/issues/8

The tests are tempfixed but I'll probably have to redo all the SSR tests....

### CODE REVIEW

File: src/dispatcher/index.ts [hunk 1]
```diff
@@ -192,37 +191,61 @@ const createDispatcher = () => {
...
+
+      // @ts-ignore
+      const metas = [...metaQueue].filter((meta) => {
+        if (!visited.has(meta.charset ? meta.keyword : meta[meta.keyword])) {
+          visited.add(meta.charset ? meta.keyword : meta[meta.keyword]);
+          return true;
+        }
+      });
 
       const stringified = `
```
Comment (JoviDeCroock) [inline src/dispatcher/index.ts:213]: Contemplating on removing the `stringified`

Comment (gustavpursche) [reply src/dispatcher/index.ts:213]: I agree. :+1:

**Comment by JoviDeCroock** [comment]
@gustavpursche that's concerning :D could you try just copying in the build files instead of a symlink to make sure it's on your machine else I'll investigate further, every part of new code should be covered by the `!isServerSide` flag though

**Comment by gustavpursche** [comment]
@JoviDeCroock The infinite loop was unrelated :+1:

**Pull request merged**